sdk/webpubsub-socketio-extension/examples/chat-with-auth-github/Readme.md

```diff
@@ -0,0 +1,44 @@
+# Create a chat app with Web PubSub for Socket.IO And GitHub OAuth
+
+## Prerequisites
```

Comment:
is this chat app with GitHub OAuth referencing any tutorial? Do you want to mention what are the lines updated specifically for Web PubSub?